src/EnergyPlus/PluginManager.cc

```diff
@@ -541,6 +542,21 @@ PluginManager::PluginManager(EnergyPlusData &state) : eplusRunningViaPythonAPI(s
                 // nothing to do here
             }
         }
+    } else {
+        // if no search path objects exist, we still need to do the default searching
+        PluginManager::addToPythonPath(state, ".", false);
```

Comment:
Also search the default locations in the case where the user did not use the SearchPaths object.